apiParser/src/Program.cs

```diff
@@ -34,6 +34,7 @@ public static class Program
 
```

Comment:
@citizenmatt should I do something with that list?

Reply:
Nah. It's useful if you need to completely recreate the docs list, but you need ALL of those folders installed somewhere, and that's several gigabytes of html. It's much easier just to incrementally update it by comparing the existing API list against a new docs folder. Leave this around if we ever have a need to completely rebuild. I know we probably won't, and it's in source control and everything, but, I can't get too worked up about an internal tool :)